server/re/include/irods/reAction.hpp

```diff
@@ -64,6 +64,7 @@ int msiCommit( ruleExecInfo_t *rei );
 int msiDeleteCollByAdmin( msParam_t *parColl, msParam_t *childName, ruleExecInfo_t *rei );
 int msiDeleteUser( ruleExecInfo_t *rei );
 int msiAddUserToGroup( msParam_t *msParam, ruleExecInfo_t *rei );
+int msiRemoveUserFromGroup(MsParam* _group, MsParam* _user, MsParam* _user_zone, RuleExecInfo* _rei);
```

Comment:
i notice these parameters are not consistent with the ones around it... others are all `_t`...  can you explain why?

Reply:
`MsParam` is the concrete type name while `msParam_t` is a type alias for `MsParam`.

They are identical to the implementation. `msParam_t` exists because of C.

Reply:
gotcha - so we *could* update all the others if we wanted to 'modernize'...   but not today.

Reply:
That's correct.

Modernize is probably not the right word for this since `msParam_t` isn't outdated. I just chose to use that today. Could change my mind tomorrow for any number of reasons :-)

Reply:
Do you have a preference for one over the other?

Reply:
no, defer to you and team.   just caught my eye is all.

Reply:
Leaving as is. This isn't urgent at the moment.